VORPCORE/vorp_inventory-lua
# Add desc for item, currency, weapon | add gold like dollars | add some feature for bank (use bank like stash) | better count item

# Inventory in Lua System for VORPCore framework

## Requirements
- [VORP Core](https://github.com/VORPCORE/VORP-Core/releases)
- [VORP Inputs](https://github.com/VORPCORE/VORP-Inputs/releases)
- [VORP Character](https://github.com/VORPCORE/VORP-Character/releases)

## Original File 
- [VORP Inventory](https://github.com/VORPCORE/vorp_inventory-lua)

## How to install
* Download the lastest version of VORP Inventory
* Copy and paste ```vorp_inventory``` folder to ```resources/vorp_inventory```
* Add ```ensure vorp_inventory``` to your ```resource.cfg``` file
* To change the language go to ```resources/vorp_inventory/Config``` and change the default language, also you will have to edit the html file to change the text on the inventory menu


## Features
* Unique weapons in order not to duplicate them.
* Each weapon has its own ammo and can have diferent type of ammo.
* Each weapon has its own modifications. (:warning: Not implemented yet)
* When dropping or giving a weapon you give it with all the modifications and ammo.
* It also has usable items.
* KLS.

## Extra Features
* All features from vorp_inventory_lua 1.0.7
* Description of all items in DB
* Gold item like Dollars (You can give and drop item)

![image](https://i.imgur.com/iJ7Baka.png)
<img width="354" alt="image" src="https://i.imgur.com/WVXkdJu.png">
<img width="354" alt="image" src="https://i.imgur.com/ZS6BjWI.png">


## Wiki
[Wiki VORP Inventory](http://docs.vorpcore.com:3000/vorp-inventory)

## Credits
- To [Val3ro](https://github.com/Val3ro) for the initial work.
- to [Emolitt](https://github.com/Emolitt) and [Outsider](https://github.com/outsider31000) for finishing/testing.   
- Credits to Vorp Team for creating the C# version and [Local9](https://github.com/Local9).
